my-app/src/components/Docs/Inputs.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect, Fragment } from "react";
+import { Table, Button } from 'semantic-ui-react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTrashAlt } from '@fortawesome/free-solid-svg-icons'
+import { faEdit } from '@fortawesome/free-solid-svg-icons'
+
+
+const ContactTableRow = props => {
+  const [data, setData] = useState(props.currentData);
+
+  useEffect(() => {
+    setData(props.currentData);
+  }, [props]);
```

Comment:
Can we use here `props.currentData` as dependencies table for this effect? 
I see it only relies on `props.currentData`, please verify